### PR TITLE
No bug, use in-tree action to group tsc checks output

### PR DESCRIPTION
**Pull request by Pike**
I wanted to make it easier to see the numbers of errors in a PR, and thought this could be done in two ways:

Add an extra log group for the tsc invocation, which collapses the full log. You can still expand the twisty if you want to know the details. It repeats the error count afterwards.

Collect the error count as step output, and thus enabling a summary step that just prints all the numbers together.

The best way to do that that I could come up with was to actually create an action to wrap tsc invocations.

### CODE REVIEW

File: .github/actions/check-tsc/index.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const { promisify } = require('util');
+const {exec} = require('child_process');
```
Comment (eemeli) [inline .github/actions/check-tsc/index.js:2]: Should probably run Prettier on this file?

File: .github/actions/check-tsc/package.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+  "name": "check-tsc",
+  "version": "0.0.0",
```
Comment (eemeli) [inline .github/actions/check-tsc/package.json:3]:
```suggestion
```
With `private: true` this shouldn't be required.

**Review by eemeli — APPROVED** [review]
Looks good, though a bit overengineered. Added a couple of nitpicks as line comments.

**Comment by Pike** [comment]
True story, I actually de-engineered this quite a bit. Not that it ever was in TypeScript, but it was close to being so a few times. And it did have external dependencies for exec and console.log 💣 .

The other part of me was tempted to put this into an exercise to learn PowerShell.

And yes, I started with bash, and failed, and then overengineered it. And then stripped it back down. Who cares about the order of stdout and stderr anyway, right?

Off to get some help.

**Pull request merged**